applications/rag/workloads.tfvars

```diff
@@ -44,15 +43,36 @@ jupyter_service_account        = "jupyter-system-account"
 dataset_embeddings_table_name = "googlemaps_reviews_db"
 
 ## IAP config
-add_auth                = false # Set to true when using auth with IAP
 brand                   = "projects/<prj-number>/brands/<prj-number>"
-support_email           = "<email>"
-k8s_ingress_name          = "jupyter-ingress"
-k8s_backend_config_name   = "jupyter-iap-config"
-k8s_backend_service_name  = "proxy-public"
-
-url_domain_addr   = ""
-url_domain_name   = ""
-client_id         = ""
-client_secret     = ""
-members_allowlist = ["allAuthenticatedUsers", "user:<email>"]
+
+## Jupyter IAP Settings
+jupyter_add_auth                  = true # Set to true when using auth with IAP
```

Comment:
when we say enable IAP on rag.. that bydefault means that jupyter and rag frontend .. both needs same IAP configs.. like add_auth, support_email,

Reply:
I think add auth and support email can be different. Customer can only enable the jupyter or frontend, and the support team for those two can be different